### ISSUE TITLE
Too many dots in version number.

### ISSUE THREAD

**Issue by barryrowlingson** [chrism0dwk/savsnet-geoaggregate#2]
The pyproject.toml file has:

`version = "0.1.1-alpha.0"`

and then if `version.py _version()` function gets called and gets the version from `_get_version_from_pyproject()` then it gets:

`"0.1.1-alpha.0"`

which gets split on line 20 into:

`['0', '1', '1-alpha', '0']`

which gets unpacked on line 23 into:

```
MAJOR, MINOR, PATCH = ['0', '1', '1-alpha', '0']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 3)
```

**Comment by chrism0dwk** [comment]
Fixed in b56d90d.  Can you confirm this now runs as intended?

**Comment by chrism0dwk** [comment]
Updated in 370801d to a more parsimonious use of code.  Again, confirm, pse.
(next time I'll open a merge request)
